.eslintrc

```diff
@@ -260,6 +260,6 @@
     "import/no-duplicates": 2,
     "import/newline-after-import": 2,
     "import/unambiguous": 2,
-    "import/extensions": [ 1, "always" ]
+    "import/extensions": [ 0, "always" ]
```

Comment:
This passes travis so there must be some cause specific to your machine.

Are you running the lint command or are you using your editors linter?